parser.y

```diff
@@ -6,7 +6,7 @@ class Lrama::Parser
 
 rule
 
-  input: prologue_declaration* bison_declaration* "%%" rules_or_grammar_declaration+ epilogue_declaration?
+  input: prologue_declaration* bison_declaration* "%%" (rules_or_grammar_declaration+ epilogue_declaration?)?
```

Comment:
Could you keep at least one `rules_or_grammar_declaration` is required then do not change the grammar on this PR?
"lib/lrama/grammar/stdlib.y" can be valid if `%%` on line 142 is moved to line 10 or around because `%rule ...` can be both `bison_declaration` and `rules_or_grammar_declaration` then they are interpreted as `rules_or_grammar_declaration`.

Reply:
ok. I revert it.